incubator/fluentd-cloudwatch/templates/daemonset.yaml

```diff
@@ -32,11 +32,11 @@ spec:
               mountPath: /config-volume
             - name: config
               mountPath: /etc/fluentd
+      hostNetwork: {{ default false .Values.hostNetwork }}
```

Comment:
Yes it should be [spec.hostNetwork](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#pod-v1-core)

It's better to uncomment the `# hostNetwork: false`  line in `values.yaml` than to add a default here (doing both is redundant). The `values.yaml` declaration is more clear to users about what they can override.

Reply:
Uncommented hostNetwork: false in values.yaml